src/style.css

```
@@ -0,0 +1,46 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 22px;
+  width: 140px;
+  height: 160px;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars--1 div:nth-child(1) {
```

```suggestion
.stars--1 stars__star:nth-child(1) {
```
У этого элемента есть класс

---

Выдает ошибку:
Expected selector ".stars__star" to come before selector          no-descending-specificity
          ".stars--1 .stars__star:nth-child(1)"